src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { Loader } from '../components/Loader';
+import { LoadingError, NoPeopleMessage } from '../components/LoadingError';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [error, setError] = useState(false);
+
+  const uploadPeople = async () => {
+    try {
+      const data = await getPeople();
+
+      const visiblePeople = data.map(person => ({
+        ...person,
+        mother: data.find(
+          mother => mother.name === person.motherName,
+        ),
+        father: data.find(
+          father => father.name === person.fatherName,
+        ),
+      }));
+
+      setPeople(visiblePeople);
+    } catch (err) {
+      setError(true);
+    }
+  };
+
+  uploadPeople();
```

Comment:
load it on the first render
open network. You sent a request to the server each time when state updates 
<img width="1425" alt="Screenshot 2022-10-07 at 01 48 22" src="https://user-images.githubusercontent.com/47606379/194432394-67bab0fa-3275-4254-9a88-fdf4180bc39e.png">